src/nina_vesseltun_ex/ex02/message_entropy.py

```diff
@@ -0,0 +1,27 @@
+import math
```

Comment:
You may only use the log2 function in this package, so it would be better 
`from math import log2`